devhelp/INSTALL

```diff
@@ -0,0 +1,365 @@
+Installation Instructions
```

Comment:
This file is always not very useful, these are just generic install instructions which mainly don't fit for this plugin.
I'd say better remove it than just keeping it around.

Reply:
Done.